src/server.ts

```diff
@@ -28,6 +33,20 @@ const exposeDocs = true
   if (isMultitenant) {
     await runMultitenantMigrations()
     await listenForTenantUpdate(PubSub)
+
+    runMigrationsOnAllTenants()
```

Comment:
on server start up, all storage api instance will race to acquire the lock.
The one who acquired the lock will create the jobs for running the migrations, the other instances who failed will simply return and do nothing.